graph/schema.graphqls

```diff
@@ -4,6 +4,7 @@ extend type Mutation {
   # normalizeTest accepts test ID (part of a test run) and updates the respective testcase
   # with the responses from the test result
   normalizeTest(id: String!): Boolean!
```

Comment:
we should remove the old normalize method, it doesn't make sense to have it anymore.